src/Classifiers/ControllerClassifier.php

```diff
@@ -13,6 +15,21 @@ public function getName()
 
     public function satisfies(ReflectionClass $class)
     {
+        return $this->extendsBaseController($class) || $this->hasBeenRegisteredInARouter($class);
+    }
+
+    protected function extendsBaseController($class)
```

Comment:
Is there any actual need to perform this check? The second one will detect all controllers.

Reply:
I think the `extendsBaseController` check is much faster the going through the RoutesCollection (like if you have more than 200 routes)

Reply:
The thing is, `extendsBaseController` will detect even controllers that are not using within the application while `hasBeenRegisteredInARouter` will only get the one that are actually used.

Plus, I hardly think that performance would matter here.

Reply:
Then the questions would be, should the `stats` command detect all Controllers or only those which are registered with a route. 
But thinking about it, I could not imagine a project where you would have a lot of Controllers which are not associated with any route.

will remove the `extendsBaseController` method.